frontend/src/app/shared/form/oib-checkbox/oib-checkbox.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, Input } from '@angular/core';
+import { formDirectives } from '../../form-directives';
+import { OibCheckboxFormControl } from '../../../model/form.model';
+import { NgForOf, NgIf } from '@angular/common';
+import { ControlContainer, FormGroupName } from '@angular/forms';
+
+@Component({
+  selector: 'oib-checkbox',
+  standalone: true,
+  imports: [...formDirectives, NgIf, NgForOf],
+  templateUrl: './oib-checkbox.component.html',
+  styleUrls: ['./oib-checkbox.component.scss'],
+  viewProviders: [
+    {
+      provide: ControlContainer,
+      useExisting: FormGroupName
+    }
+  ]
+})
+export class OibCheckboxComponent {
```

Comment:
shouldn't that be a ControlValueAccessor?
That would avoid passing the settings as the input

Reply:
Same for all other components